main/.vitepress/themeConfig/nav.js

```diff
@@ -38,6 +38,14 @@ export const nav = [
         ariaLabel: 'Agoric CLI',
         link: '/guides/agoric-cli/',
       },
+      {
+        text: 'Agoric API Reference',
```

Comment:
This entry seems apt to me but open to suggestion if we wanna call it something else